src/Map/OpenGL/OpenGLES2plus/GPUAPI_OpenGLES2plus.cpp

```diff
@@ -1869,3 +1871,66 @@ bool OsmAnd::GPUAPI_OpenGLES2plus::pickFramebufferDepthValue(
 
     return true;
 }
+
+void OsmAnd::GPUAPI_OpenGLES2plus::enableOffscreenRendering(const GLsizei bufferWidth, const GLsizei bufferHeight)
+{
+    const bool alreadyEnabled = _offscreenFramebuffer != 0;
+    if (alreadyEnabled)
+        return;
+
+    glGenFramebuffers(1, &_offscreenFramebuffer);
+    GL_CHECK_RESULT;
+    glBindFramebuffer(GL_FRAMEBUFFER, _offscreenFramebuffer);
+    GL_CHECK_RESULT;
+
+    glGenRenderbuffers(1, &_colorRenderbuffer);
+    GL_CHECK_RESULT;
+    glBindRenderbuffer(GL_RENDERBUFFER, _colorRenderbuffer);
+    GL_CHECK_RESULT;
+
+    glRenderbufferStorage(GL_RENDERBUFFER, GL_RGBA4, bufferWidth, bufferHeight);
+    GL_CHECK_RESULT;
+
+    glFramebufferRenderbuffer(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_RENDERBUFFER, _colorRenderbuffer);
```

Comment:
`glFramebufferRenderbuffer ` is not supported in GLES 2.0 so you're actually using GLES 3.0 features w/o version check

Reply:
Why not? There are two pages ([1](https://docs.gl/es2/glFramebufferRenderbuffer), [2](https://registry.khronos.org/OpenGL-Refpages/es3.0/html/glFramebufferRenderbuffer.xhtml)) about `glFramebufferRenderbuffer`, in both GLES 2.0 is marked as supported.

Reply:
Ah, true, I stand corrected. Please disregard my comment 🙏